addons/OPT/REVIVE/fn_dialog.sqf

```diff
@@ -0,0 +1,174 @@
+/**
+* Description:
+* Dialogsteuerung 
+*
+* Author:
+* Lord-MDB
+*
+* Arguments:
+* None
+*
+* Return Value:
+* None
+*
+* Server only:
+* No
+*
+* Public:
+* No 
+* 
+* Global:
+* No
+* 
+* API:
+* No
+*
+* Example:
+* [] call FUNC(dialog);
+*/
+#include "macros.hpp"
+
+//Dialog erstellen
+private _success = createDialog "opt_revive_blackscreen"; 
+
+//Dialog definieren
+#define IDD_REVIVE_BLACKSCREEN 5000
+#define IDC_REVIVE_BUTTON 5011
+#define IDC_REVIVE_MEDICNEARLABEL 5012
+#define IDC_REVIVE_MEDICNEARLABEL_METER 5013
+#define IDC_REVIVE_BLEEDOUTBAR 5014
+#define IDC_REVIVE_BLEEDOUTBAR_TEXT 5015
+
+private _display = findDisplay IDD_REVIVE_BLACKSCREEN;
+private _BleedoutBar = _display displayCtrl IDC_REVIVE_BLEEDOUTBAR;
+private _Respawn_button = _display displayCtrl IDC_REVIVE_BUTTON;
+private _MedicNearLabel_Meter = _display displayCtrl IDC_REVIVE_MEDICNEARLABEL_METER;
+private _BleedoutBar_Text = _display displayCtrl IDC_REVIVE_BLEEDOUTBAR_TEXT;
+
+//Grundstellung
+_BleedoutBar progressSetPosition 1.0; 
+_MedicNearLabel_Meter ctrlSetText "";  
+_BleedoutBar_Text ctrlSetText format ["%1 sec",GVAR(ausblutzeit)]; 
+
+//Chat abschalten
+1 enableChannel false;
+
+// Respwan Button
+_Respawn_button ctrlAddEventHandler [ "ButtonClick", 
+{
+	player setDamage 1;
+	1 enableChannel true;
+	player allowDamage true;
+
+}];
+
+GVAR(startzeit) = time;
```

Comment:
Sollte nicht in einer globalen Variable gehalten werden, sondern als Parameter an den PFH übergeben werden.